gammapy/makers/tests/test_utils.py

```diff
@@ -213,6 +215,7 @@ def make_map_background_irf_with_symmetry(fpi, symmetry="constant"):
         ontime="42 s",
         bkg=bkg_3d_custom(symmetry),
         geom=WcsGeom.create(npix=(3, 3), binsz=4, axes=[axis], skydir=fpi.fixed_icrs),
+        fov_rotation_error_limit=0.5 * u.deg,
```

Comment:
The function `make_map_background_irf` does not have a default value for `fov_rotation_error_limit`.
Only the class `MapDatasetMaker` has an associated default.

I would have preferred giving  `make_map_background_irf` the default and overwrite it with the value passed to `MapDatasetMaker` only if it is provided. But I didn't find an elegant way to do so.
Alternatively, I could give both a default value but I am worried it could lead to confusion later.